mindsdb/integrations/redis/redisdb.py

```diff
@@ -1,143 +1,42 @@
-from threading import Thread
 import walrus
-from mindsdb.integrations.base import StreamIntegration
-from mindsdb.streams.redis.redis_stream import RedisStream
-from mindsdb.streams.base.base_stream import StreamTypes
 
-from mindsdb.interfaces.storage.db import session, Stream
-from mindsdb.interfaces.database.integrations import get_db_integration
+from mindsdb.integrations.base import StreamIntegration
+import mindsdb.interfaces.storage.db as db
+from mindsdb.streams import RedisStream, StreamController
 
 
 class RedisConnectionChecker:
-    def __init__(self, **kwargs):
-        self.connection_info = kwargs.get("connection", {})
-        self.advanced_info = kwargs.get("advanced", {})
-        self.connection_params = {}
-        self.connection_params.update(self.connection_info)
-        self.connection_params.update(self.advanced_info)
-
-    def _get_connection(self):
-        return walrus.Database(**self.connection_params)
+    def __init__(self, **params):
+        self.connection_info = {
+            'host': params['connection']['host'],
+            'port': params['connection']['port'],
+            'password': params['connection']['password'],
+        }
 
     def check_connection(self):
-        client = self._get_connection()
         try:
+            client = walrus.Database(**self.connection_info)
```

Comment:
It may be false-positive: for e.g connection might be successfully established, but you can't do anything because of lack of permissions (as example empty or wrong password)
```
import walrus
CONNECTION_PARAMS = {"host": "3.220.66.106", "port": 6379, "db": 0, "password": "WRONG_PASS"}
client = walrus.Database(**CONNECTION_PARAMS)
```

Reply:
so `client.dbsize()` is not enough to check the connection?